tests/test_schema.py

```diff
@@ -1150,12 +1194,10 @@ class World(Base):
 
         world_schema = SQLAlchemySchemaNode(World)
 
-        self.assertTrue(
-            isinstance(world_schema['not_a_number'].typ, colander.String))
+        self.assertIsInstance(
```

Comment:
ah.. nevermind.  It's because we're importing `unittest2` in earlier versions which imports the new methods.  Carry on!  ^_^